package.json

```diff
@@ -20,9 +20,12 @@
     "dist/*.js"
   ],
   "devDependencies": {
+    "rollup-plugin-babel": "^4.4.0",
+    "rollup-plugin-cleanup": "^3.1.1",
     "@rollup/plugin-commonjs": "^11.0.2",
     "@rollup/plugin-node-resolve": "^7.1.1",
-    "chart.js": "^2.8.0",
+    "babel-plugin-istanbul": "^6.0.0",
+    "chart.js": "^3.0.0-alpha",
```

Comment:
I'd avoid updating the chart.js version for now so that the tests will pass and we can merge this without waiting for a final release of Chart.js 3.0